src/z3c/etestbrowser/wsgi.txt

```diff
@@ -1,8 +1,9 @@
-Extended testbrowser for zope.testbrowser.wsgi
-----------------------------------------------
+Deprecated special support for zope.testbrowser.wsgi
+----------------------------------------------------
 
-There is also a variant in ``z3c.etestbrowser.wsgi`` which can be used for
-the WSGI variant of ``zope.testbrowser``.
+There was also a variant in ``z3c.etestbrowser.wsgi`` which could be used for
+the WSGI variant of ``zope.testbrowser``. It is no longer necessary because.
+``z3c.etestbrowser.testing`` now speaks WSGI.
 
 Example:
```

Comment:
This basically tests the deprecation, which is fine, and will go with 4.0. I wonder, whether we should place a hint somewhere to remove that, after the 3.0 Release.

Reply:
I created #13 and will add a note about the planned removal.